Code-the-Dream-School/dd-prac-team1-front
# Devops/render saved recipe(PRA-99)

## One Line Description
setup GET function and render saved recipes on page

## Requirements

_What this code should do_
render previously saved im MongoDB recipes on "saved recipe" page

## Notes

_Any additional info_

## Test Steps

_Instructions on how to test the changes_
login - create recipe with AI - save recipe - click "SAVED" navbar button - you should see all saved recipes on "SAVED RECIPE" page 

## UI/UX Outcome

_Attach a screenshot here_
<img width="1378" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/45f37ec7-85e2-4b97-a4d7-aa535428d57d">

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
